### DISCUSSION TITLE
Tailwind table not styled correctly

### DISCUSSION THREAD

**Discussion by aznanjamil** [Power-Components/livewire-powergrid#652] [Q&A]
![Screenshot 2022-10-05 at 10 04 54 AM](https://user-images.githubusercontent.com/47454846/193963706-8a5dbece-07b2-43b7-91ad-bee93ee66b84.png)
Hi, i have this issue that the table is not style correctly. Im using the tailwind theme and followed all of the required configuration. Any idea why is this happening ?

**Comment by felixmpa** [comment]
> 
It happened to me when I installed it the first time, check the following steps:

resources/css/app.css

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

app.blade.php 
```
<head>
//.... include after livewire
@livewireStyles
powerGridStyles
</head>
...........
@livewireScripts
@livewire('livewire-ui-modal')
@powerGridScripts
</body>
```

package.json
```
in your devDependencies be sure have: @tailwindcss/forms
```

And of course, be sure to [run npm dev] and clear all view:cache again.

**Comment by luanfreitasdev** [comment]
https://livewire-powergrid.com/#/get-started/configure?id=_5-tailwind-settings
